src/test/kotlin/racing/domain/CapacityTest.kt

```diff
@@ -0,0 +1,56 @@
+package racing.domain
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.comparables.shouldBeGreaterThan
+import io.kotest.matchers.comparables.shouldBeLessThan
+import io.kotest.matchers.shouldBe
+import java.lang.IllegalArgumentException
+
+class CapacityTest : BehaviorSpec({
+    given("Capacity 객체 생성자는") {
+        `when`("숫자를 인자로 전달하면") {
+            val actual = Capacity(1)
+
+            then("정상적으로 생성된다") {
+                actual.value shouldBe 1
+            }
+        }
```

Comment:
Kotest의 BehaviorSpec을 사용해주셨네요👍👍👍
불행히도 `when`은 예약어라 when절은 벡틱 문자로 감싸진게 저는 조금 불편하더라구요🥹
다행히 다음과 같이 대문자로 각 구문을 표현하면 벡틱 없이도 When 절을 사용할 수 있습니다!
```suggestion
    Given("ggg") {
        When("www") {
            Then("ttt") {
                // **
            }
        }
    }
```

Reply:
오.... 좋은 꿀팁이네요 감사합니다 ㅎㅎ